src/comonents/InputForm.jsx

```diff
@@ -1,11 +1,21 @@
-import React from 'react'
+import React, { useContext } from "react";
+import ButtonPage from "./ButtonPage";
+import { MyNoteContext } from "../context/NoteState";
 
 function InputForm() {
```

Comment:
this component name should be InputField, not form,
this represent particular input field.

Reply:
Changed